README.md

```diff
@@ -149,6 +149,18 @@ A command line flag exists `--podCheckNamespaces` which can optionally contain a
 - Error state toleration: 5 minutes
 - Check name: `podStatus`
 
+#### DNS
+
+Checks for failures with DNS, including resolving within the cluster and outside of the cluster. Default endpoints to resolve: kubernetes.default, aws.amazon.com, cloud.dns.com
+
+A command-line flag exists `--dnsEndpoints` which can optionally include a comma separated list of DNS endpoints to test. 
+
+- Timeout: 1 minutes
+- Check Interval: 2 minutes
+- Error state toleration: 5 minutes
```

Comment:
I think you removed this five minute state toleration, correct?

Reply:
Updated it to be a 1 minute toleration

Reply:
Why have any time toleration?  I would assume that any dns failure against kubernetes.default would be an error.  Maybe give it a second query on failure to tolerate small hiccups, but failures should be rare and are probably bad.

What do you think?

Reply:
I mentioned this below, but I agree.  I'd like to run a re-try or a faster interval to verify that the DNS is consistently down before alerting on the state.  Optionally, we could do something like the podRestartChecker and compare DNS responses over time and only alert once some threshold of failures is breached.